profiles/sssd/password-auth

```diff
@@ -20,11 +21,13 @@ account     sufficient                                   pam_localuser.so
 account     sufficient                                   pam_usertype.so issystem
 account     [default=bad success=ok user_unknown=ignore] pam_sss.so
 account     required                                     pam_permit.so
+-account    sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
```

Comment:
pam_permit should remain last in this stack. 

Place this imediately before pam_unix here.

Reply:
Done